site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -146,69 +184,251 @@ def _transform_publication_date(self):
 
     def _transform_document_type(self):
         """Mapping of INSPIRE document type to resource type."""
-        document_type = self.inspire_metadata.get("document_type")[0]
-        document_type_mapping = {
-            "activity report": "publication-report",
-            "article": "publication-article",
-            "book": "publication-book",
-            "book chapter": "publication-section",
-            "conference paper": "publication-conferencepaper",
-            "note": "publication-technicalnote",
-            "proceedings": "publication-conferenceproceeding",
-            "report": "publication-report",
-            "thesis": "publication-thesis",
-        }
+        # document_type = self.inspire_metadata.get("document_type")[0]
+        #
+        # document_type_mapping = {
+        #     "activity report": "publication-report",
+        #     "article": "publication-article",
+        #     "book": "publication-book",
+        #     "book chapter": "publication-section",
+        #     "conference paper": "publication-conferencepaper",
+        #     "note": "publication-technicalnote",
+        #     "proceedings": "publication-conferenceproceeding",
+        #     "report": "publication-report",
+        #     "thesis": "publication-thesis",
+        # }
+        #
+        # if document_type not in document_type_mapping:
+        #     self.metadata_errors.append(
+        #         f"Error occurred while mapping document_type to resource_type. Couldn't fine a mapping rule for "
+        #         f"document_type {document_type}. INSPIRE record id: {self.inspire_metadata.get('control_number')}."
+        #     )
+        #     return None
+
+        # return document_type_mapping.get(document_type)
+
+        document_types = self.inspire_metadata.get("document_type", [])
+        for doc_type in document_types:
+            if doc_type == "article":
+                self.metadata_errors.append("Articles are not supported for now.")
+                return None
```

Comment:
we should check in a demo what happens with multiple types document import

Reply:
Sure! I have also added a test for the case if a record has 2 doc types and one of them is article: `"document_type": ["thesis", "article"]`. It fails with an error in that case. See here: https://github.com/CERNDocumentServer/cds-rdm/pull/444/files#diff-fbdc1c27597a8be737b958bc042fdb7ea4b30e03a308f7b564b3705804b63f72R672